components/Helpers/src/NativeMethods.json

```diff
@@ -0,0 +1,3 @@
+{
+  "allowMarshaling":  false
```

Comment:
Add scheme and preserve sig.
```suggestion
  "$schema": "https://aka.ms/CsWin32.schema.json",
  "allowMarshaling": false,
  "comInterop": {
    "preserveSigMethods": [
      "*"
    ]
  }
```

Reply:
We only use
```
RegisterClassEx
DefWindowProc
CreateWindowEx
WM_SETTINGCHANGE
```

Not a single COM interface gets imported.
So I don't think comInterop.preserveSigMethods is needed.

Reply:
Contributors may use com interfaces for other things. There's no reason to keep it out. Also add the schema.

Reply:
@0x5bfa Done